packages/gatsby/src/bootstrap/index.d.ts

```diff
@@ -0,0 +1,5 @@
+type BootstrapArgs = {
```

Comment:
I think this should be an `interface` instead of a `type` in this case?

Reply:
I generally agree. I'll make these changes